compliance/rio/src/test/java/org/eclipse/rdf4j/rio/helpers/RDFParserHelperTest.java

```diff
@@ -108,7 +108,7 @@ public final void testCreateLiteralLabelOnly()
 				valueFactory);
 
 		assertEquals(LABEL_TESTA, literal.getLabel());
-		assertFalse(literal.getLanguage().isPresent());
+		assertFalse(literal.getLanguage() != null);
```

Comment:
If this were a long term branch (does it have an end of support date yet?), this pattern may be better suited to use assertNull, as the assertFalse(... != null) pattern isn't intuitive to read. However, it is functionally the same so really just a style issue. 

Not sure what Git will think about this change if/when you merge 1.0 into 2.0. Ie, will it change the code on 2.0 to use != null which will always be true for Optional's or will it conflict. May be simpler not to merge 1.0 to 2.0 and just recreate patches as necessary.